### ISSUE TITLE
Scaffold app with eslint-config-loopback

### ISSUE THREAD

**Issue by superkhau** [strongloop/loopback-workspace#310]
See https://github.com/strongloop/loopback-example-session/commit/23dc62c9eef03c951dde01182bde0b4191885a59#commitcomment-18571031

- [x] Fix workspace template
- Fix at https://github.com/strongloop/loopback-workspace/pull/312

Fix affected repositories:
- [x] https://github.com/strongloop/loopback-example-session
- Fix at https://github.com/strongloop/loopback-example-session/pull/1
- [x] https://github.com/strongloop/loopback-sandbox
- Fix at https://github.com/strongloop/loopback-sandbox/pull/9

**Comment by bajtos** [comment]
Looks like the template is missing a dev-dependency on `eslint-config-loopback`, see https://github.com/strongloop/loopback-workspace/blob/bcb269a8a728dc521f4adbacbde2117a6527f9c9/templates/projects/empty-server/data.js#L29-L32.

**Comment by superkhau** [comment]
Another thing to note is after installing `eslint-config-loopback`, linting fails from the default scaffold with:

```
/.../loopback-example-session/server/boot/root.js
  1:1  error  Use the global form of 'use strict'  strict

/.../loopback-example-session/server/server.js
  1:1  error  Use the global form of 'use strict'  strict
```

We should fix this off the default scaffold too.

**Comment by superkhau** [comment]
I think we should scaffold based on our eslint conventions (npm run lint, posttest: eslint .) instead of pretest.

**Comment by superkhau** [comment]
Another thing I ran into is when scaffolding models, the `*/models/*.js` files will need `use strict` automatically added when we scaffold the files too via `slc loopback:model`.

**Comment by superkhau** [comment]
Fixed via strongloop/loopback-workspace#312
